tokendito/okta.py

```diff
@@ -429,7 +458,7 @@ def extract_state_token(html):
     state_token = None
     pattern = re.compile(r"var stateToken = '(?P<stateToken>.*)';", re.MULTILINE)
 
-    script = soup.find("script", text=pattern)
+    script = soup.find("script", string=pattern)
```

Comment:
⚠️ This change removes warnings that were appearing when running tests due to a deprecation.
```
/Volumes/xxxxx/tokendito/tokendito/okta.py:461: DeprecationWarning: The 'text' argument to find()-type methods is deprecated. Use 'string' instead.
  script = soup.find("script", text=pattern)
```

Before:
```
191 passed, 18 deselected, 5 warnings in 0.85s
```

After:
```
191 passed, 18 deselected in 0.66s
```